### ISSUE TITLE
generalise attestation issuer code and export it as a library

### ISSUE THREAD

**Issue by colourful-land** [TokenScript/attestation#175]
1. Make sure fork from the latest main at a later point than https://github.com/TokenScript/attestation/commit/6a6781ce4bccdb86cea877776c65159538e7d5a3
2. work on `./src/main/java/org/tokenscript/attestation/IdentifierAttestation.java` to replace the  attestation type `twitter` into a labeledURI or InetPersona type.

The InetPersona type is any x.509 attestation with labeledURI as the subject, which is typically the URL of the person's online persona, followed by an immutable identifier, such as `https://twitter.com/zhangweiwu 180283019`.

This is per @zhangzhongnan928's guidance that we will soon see sites such as attestation.id to producing attestation of types other than twitter, such as deviantart.com.

Once this is done, we need to package this thing into a maven repository and let https://github.com/AlphaWallet/attestation.id source from it.

**Comment by jot2re** [comment]
I am a bit unsure of what exactly needs to be done. The constructor for `IdentifierAttestation.java` where the user's ID is public and stored as a labeledURI, already supports the format of an arbitrary URL and and arbitrary identifier.
In regard to the attestation type it is only used in the setting of attestations hiding the user's identifier. As far as I understand this is not what you want here, right?

**Comment by jot2re** [comment]
I am a bit unsure of what exactly needs to be done. The constructor for IdentifierAttestation.java where the user's ID is public and stored as a labeledURI, already supports the format of an arbitrary URL and and arbitrary identifier.
In regard to the attestation type it is only used in the setting of attestations hiding the user's identifier. As far as I understand this is not what you want here, right?

**Comment by jot2re** [comment]
In any case I have changed the implementation for hidden identifier to use `InetPersona` instead of twitter.

**Comment by colourful-land** [comment]
> In any case I have changed the implementation for hidden identifier to use `InetPersona` instead of twitter.

that's the right thing to do but you haven't submitted right? not seeing PR

**Comment by jot2re** [comment]
No, not yet. I am looking into setting up the right Maven files to deploy to Maven central as well.

**Comment by colourful-land** [comment]
Remembet to put a comment where InetPersona is and show example of twitter in comment instead of in source code.
